### ISSUE TITLE
[setup] Install Oversteer as a Flatpak

### ISSUE THREAD

**Issue by LukeShortCloud** [winesapOS/winesapOS#941]
During the first-time setup, install Oversteer as a Flatpak instead of using the AUR package (we also need to remove the package installation from the `winesapos-install.sh` script). As of version 0.8.3, it now has a Flatpak.

https://flathub.org/apps/io.github.berarma.Oversteer

**Comment by LukeShortCloud** [comment]
Related to this, CI is now failing because the desktop shortcut name for Oversteer has changed.

```
+ cp /winesapos/usr/share/applications/org.berarma.Oversteer.desktop /winesapos/home/winesap/Desktop/
cp: cannot stat '/winesapos/usr/share/applications/org.berarma.Oversteer.desktop': No such file or directory
```

I'll fix this after the 4.1.0 stable release.

**Comment by LukeShortCloud** [comment]
The new desktop shortcut for the AUR package is: `/usr/share/applications/io.github.berarma.Oversteer.desktop`.
